### PR TITLE
Redfines HoS hat as a hat and not a helmet, and slightly edits the description.

### PR DESCRIPTION
**Pull request by DimWhat**
[Balance]
## About the PR 
Removes the "blockchoke" and "coverseyes" functions of the HoS hat, and changes the definitions from a helmet to a normal hat. This makes it so the HoS can be choked and also receive CPR while wearing this hat. Also redfines all instances of /obj/item/clothing/head/helmet/HoS to a new definition of /obj/item/clothing/head/hos_hat. Finally, edits the HoS hat's descrption a little to make more sense.
From this

> "Actually, you got this hat from a fast-food restaurant, that's why it folds like it was made of paper."

to this

> "Actually, this hat is from a fast-food restaurant, that's why it folds like it was made of paper."

## Why's this needed? 
Also the HoS is unable to receive CPR nor be choked at all when wearing a hat nowhere near the neck which I find to be silly.

## Changelog

```changelog
(u)DimWhat
(+)Anyone wearing the Head of Security's hat is now able to be choked or receive CPR.
```

### CODE REVIEW

**Comment by Yellow-Mushroom** [comment]
Rip HOS’s ridiculously muscled neck, they can finally be choked like the rest of us

**Comment by Kyle2143** [comment]
No. I think that the HoS's hat is fine as an exception to this rule. It's always been this way and I think it's good to have it this way.

As an actual justification for it, I think that the HoS being such a high tier target for traitors and what have you that giving them a bit extra armor as a "base stat" in the way of the hat/beret is worthwhile. The same goes for the Captain's Hat/armor. Also, the HoS's hat itself is a high value target for traitors since it is a theft objective for them. So if it wasn't particularly good then HoS's wouldn't really have that reason to protect it and keep it out of the hands of possible enemies.

Also, I don't think that the change to the description is good. The description can be viewed by anyone, not just the HoS. So it doesn't make sense for someone to read that if the HoS gave them the beret or someone stole it from them.

**Comment by Yellow-Mushroom** [comment]
> Also, I don't think that the change to the description is good. The description can be viewed by anyone, not just the HoS. So it doesn't make sense for someone to read that if the HoS gave them the beret or someone stole it from them.

I think you misunderstood the description change. Sun changes it to where the hat’s origin is in the third person (this hat came from a restaurant) when it is currently/originally (you got this hat from a restaurant)

**Comment by DimWhat** [comment]
> No. I think that the HoS's hat is fine as an exception to this rule. It's always been this way and I think it's good to have it this way.
> 
> As an actual justification for it, I think that the HoS being such a high tier target for traitors and what have you that giving them a bit extra armor as a "base stat" in the way of the hat/beret is worthwhile. The same goes for the Captain's Hat/armor. Also, the HoS's hat itself is a high value target for traitors since it is a theft objective for them. So if it wasn't particularly good then HoS's wouldn't really have that reason to protect it and keep it out of the hands of possible enemies.
> 
> Also, I don't think that the change to the description is good. The description can be viewed by anyone, not just the HoS. So it doesn't make sense for someone to read that if the HoS gave them the beret or someone stole it from them.

I understand your points that the head of security wouldn't defend it but the head of security already has many important things tied to them such as their special ID that can open the armoury and their energy gun/lawbringer.  

As a counter to head of security giving their hat away to antagonists, the captain's hat also shares extremely similar stats to these alterations yet it is considered similar in the difficulty to antagonists as spy thiefs which gain the same level of reward to stealing either hats. 

As for the hat's description, it is in second person. I do not believe Jane Assistant got the Head of Security's hat at a fast food resturant if they stumble upon it on a dead HoS. The alternate description gimmick could be applied to the hat as well so that different roles get different examine messages though.

File: code/obj/item/clothing/hats.dm [hunk 1]
```diff
@@ -1303,3 +1303,33 @@
...
+	icon_state = "hoscap"
+	uses_multiple_icon_states = 1
+	item_state = "hoscap"
+	c_flags = SPACEWEAR
+	var/is_a_communist = 0
+	var/folds = 0
+	desc = "Actually, this hat is from a fast-food restaurant, that's why it folds like it was made of paper."
+	setupProperties()
+		..()
+		setProperty("meleeprot_head", 3)
```
Comment (Kyle2143) [inline code/obj/item/clothing/hats.dm:1319]: All other changes are fine, but the lowering of this value, without this change, it's fine to merge.

**Review by Kyle2143 — CHANGES_REQUESTED** [review]

File: code/obj/item/clothing/hats.dm (hunk 1 above)
Comment (DimWhat) [reply code/obj/item/clothing/hats.dm:1319]: You know what, I'm fine with that. This is not a hill I want to die on. Compromise and yadda yadda.

**Pull request merged**